extruct/w3cmicrodata.py

```diff
@@ -182,7 +203,8 @@ def _extract_property_value(self, node, items_seen, base_url, force=False):
             return self._extract_textContent(node)
 
     def _extract_textContent(self, node):
-        return u"".join(self._xp_clean_text(node)).strip()
```

Comment:
Looks like ``_xp_clean_text`` is not used any more, can it be removed?

Reply:
Hey @lopuhin! Didn't notice that. Sure, will remove it :+1: